stroeer/article/v1/service.proto

```diff
@@ -10,9 +10,34 @@ option java_package = "de.stroeer.article.v1";
 
 service ArticleService {
   rpc GetArticle (GetArticleRequest) returns (stroeer.core.v1.Article) {}
+
+  // Returns an Article page including all (SEO) relevant information for
+  // rendering (canonical web or AMP) an Article with the given id.
+  rpc GetArticlePage (GetArticlePageRequest) returns (ArticlePage) {}
 }
 
 message GetArticleRequest {
   string id = 1;
 }
 
+message GetArticlePageRequest {
+  string id = 1;
+}
+
+message ArticlePage {
+  stroeer.core.v1.Article main_article = 1;
+  repeated RelatedArticle related_articles = 2;
+}
+
+message RelatedArticle {
+  // we could also use an stroeer.core.v1.Article here but maybe it's worth to have a Teaser abstraction
+  // including only teaser relevant attributes?!
+  stroeer.core.v1.Teaser teaser = 1;
+  RelatedArticleType role = 2;
+}
+
+enum RelatedArticleType {
+  RELATED_ARTICLE_TYPE_UNSPECIFIED = 0;
```

Comment:
Shouldn't we just filter out relations unknown by our code during transformation?

Reply:
protobuf lint rule to always include unknown at `0`